kernel/kernel-api/src/main/java/io/delta/kernel/internal/icebergcompat/IcebergWriterCompatV1MetadataValidatorAndUpdater.java

```diff
@@ -88,6 +92,10 @@ public static void validateIcebergWriterCompatV1Change(
         throw DeltaErrors.enablingIcebergWriterCompatV1OnExistingTable(
             TableConfig.ICEBERG_WRITER_COMPAT_V1_ENABLED.getKey());
       }
+      if (wasEnabled && !isEnabled) {
```

Comment:
can the user disable icebergCompatV2?

Reply:
I think so? But you wouldn't be able to do this if icebergWriterCompatV1 was enabled

Reply:
icebergCompatV2 does allow disabling in general, but as allison notes, the icebergWriterCompatV1 checks will block disabling it if icebergWriterCompatV1 is also enabled.

Reply:
Added a test to be sure :)